sovryn-token-bridge/federator/src/lib/Globals.js

```diff
@@ -0,0 +1,5 @@
+module.exports = Object.values({
```

Comment:
This will export `[0, 0]` instead of the object -- it should probably be
```
module.exports = {
    currentEthGasBasePrice: 0,
    currentEthGasPriceAvg: 0,
};
```

Reply:
Fixed